<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewOrdersDialog</class>
 <widget class="QDialog" name="ViewOrdersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Orders</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ViewOrdersDialog {
	background-image: url(:/mc.jpg);
}

</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetOrderQueue">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>561</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0.8);
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Orders</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetOrderServed">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>411</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFront">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0.8);
border-radius: 15px;
color: white;</string>
   </property>
   <property name="text">
    <string>Next Order ...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDequeuePush">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0.8);
border-radius: 15px;
color: white;</string>
   </property>
   <property name="text">
    <string>Order #... has been served</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPeek">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0.8);
border-radius: 15px;
color: white;</string>
   </property>
   <property name="text">
    <string>Last Order served #...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDequeuePush">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>610</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: white;
border: 2px solid yellow;
font-weight: bold;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Serve the next order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPop">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>610</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: white;
border: 2px solid yellow;
font-weight: bold;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Pop Order</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
